modules/microshift-cni-customization-matrix.adoc

```diff
@@ -39,10 +39,15 @@ The following table summarizes the status of networking features and capabilitie
 |IPsec encryption for intra-cluster communication|Not available|N/A
 
 |IPv6|Not available ^[5]^|N/A
+
+|Ingress router|Yes|Yes ^[6]^
+
+|Multiple networks plug-in|Yes|Yes
 |===
 
 1. If unset, the default value is set to the next immediate subnet after the service network. For example, when the service network is `10.43.0.0/16`, the `advertiseAddress` is set to `10.44.0.0/32`.
 2. You can use the multicast DNS protocol (mDNS) to allow name resolution and service discovery within a Local Area Network (LAN) using multicast exposed on the `5353/UDP` port.
 3. There is no built-in transparent proxying of egress traffic in {microshift-short}. Egress must be manually configured.
 4. Setting up the firewalld service is supported by {op-system-ostree}.
-5. IPv6 is not available in any configuration.
+5. IPv6 is only available if using the {microshift-short} Multus plug-in. It is not directly supported in any configuration.
```

Comment:
IPv6 is a 4.17 feature, unless you are referring to something else? MicroShift does not yet support IPv6 in any of its interfaces (cluster network, service network, endpoints, router, etc.)

Reply:
My understanding is that you can add IPv6 when using a plugin via Multus that supports it.

Reply:
Yeah here it is https://github.com/openshift/openshift-docs/pull/75027/files#diff-fc92a72569f4989aca2c074d754dfdad8e3205f54eb6c046582d9b221591f66eR82

Reply:
> IPv6 is a 4.17 feature, unless you are referring to something else? MicroShift does not yet support IPv6 in any of its interfaces (cluster network, service network, endpoints, router, etc.)

I can say, "MicroShift does not support IPv6 in any of its interfaces such as the cluster network, service network, endpoints, and router."  instead of "It is not directly supported in any configuration." WDYT?

> IPv6 is a 4.17 feature, unless you are referring to something else? "MicroShift does not yet support IPv6 in any of its interfaces (cluster network, service network, endpoints, router, etc.)"

Reply:
That's fine. If you wanted to be more terse and just say "IPv6 is not supported." that would be fine too.

Reply:
Went both, if the edit makes sense to you @jerpeter1 . I want potential customers to know that IPv6 _can_ be used in some way. I think if we just leave it as simply unsupported, we lose important detail and the docs appear to conflict.